Source/ASNetworkImageNode.h

```diff
@@ -133,6 +133,13 @@ NS_ASSUME_NONNULL_BEGIN
  */
 @property (readonly) CGFloat renderedImageQuality;
 
+/**
+ * Loading progress of the current image.
```

Comment:
Let's call it "download progress" to be a bit clearer and consistent with PINRemoteImage.

```suggestion
 * Download progress of the current image.
```